Cant remove/uninstall in Fedora 37

How to remove/uninstall this project using procedure:
```
git clone https://github.com/robiot/xclicker.git
cd xclicker
make install

```

I'm trying to remove it with `make uninstall` and I have the error `make: *** No rule to process the 'uninstall' target`, I've tried to remove it in other ways using commands to remove the directory completely but without success.

--- a/Makefile
+++ b/Makefile
@@ -50,6 +50,13 @@ install: release
 	@sudo install -Dm 644 ./assets/icon.png /usr/share/pixmaps/${BINNAME}.png
 	@echo "Installed XClicker"
 
+.PHONY: uninstall
+uninstall:
+	@sudo rm /usr/bin/${BINNAME}
+	@sudo rm /usr/share/applications/xclicker.desktop
+	@sudo rm /usr/share/pixmaps/${BINNAME}.png
+	@echo "Uninstalled XClicker :("
+
 .PHONY: deb
 deb: release
 	@rm -rf ${debpkgdir}

